src/store.js

```diff
@@ -0,0 +1,13 @@
+import { configureStore } from '@reduxjs/toolkit';
+import categoriesReducer from './redux/categories/categoriesSlice';
+
+import booksReducer from './redux/books/booksSlice';
+
+const store = configureStore({
+  reducer: {
+    categories: categoriesReducer,
+    books: booksReducer,
+  },
+});
+
```

Comment:
- [ OPTIONAL ] I think it would be great if you add `store.js` file as well in redux folder since it is the store of the redux reducers so I think it would be better if they are in the same folder to have organized codebase and files 💪🏻⭐🔥